core/spec/models/spree/stock/estimator_spec.rb

```diff
@@ -140,6 +140,32 @@ module Stock
             expect(shipping_rates.first.tax_rate).to eq(tax_rate)
           end
         end
+
+        it 'uses the configured shipping rate selector' do
+          class Spree::Stock::TestSelector; end;
+          Spree::Config[:shipping_rate_selector_class] = Spree::Stock::TestSelector
+
+          shipping_rate = double(:shipping_rate, "selected=" => nil)
+          selector = double(:selector, find_default: shipping_rate)
+          allow(Spree::Stock::TestSelector).to receive(:new).and_return(selector)
+
+          subject.shipping_rates(package)
+
+          expect(selector).to have_received(:find_default)
+          expect(shipping_rate).to have_received("selected=").with(true)
+        end
+
+        it 'uses the configured shipping rate sorter' do
+          class Spree::Stock::TestSorter; end;
+          Spree::Config[:shipping_rate_sorter_class] = Spree::Stock::TestSorter
```

Comment:
this gets reset before every spec run, see https://github.com/solidusio/solidus/blob/master/core/spec/spec_helper.rb#L63